### PR TITLE
Add Sponsored Content

### PR DESCRIPTION
**Pull request by joshcosta**
Waiting on the sponsor to get back to us about the wording before this PR can be merged.

### CODE REVIEW

File: src/app/semester-home/semester-home.component.html [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
 		</mat-grid-tile>
-		<mat-grid-tile>
+		<mat-grid-tile rowspan="3">
 			<app-home-carousel></app-home-carousel>
 		</mat-grid-tile>
+		<mat-grid-tile rowspan="1">
+			<mat-card class="base-text support-container">
+				<div class="support-header">
+					<span class="support-word">SPONSORED</span>
+					<mat-icon (click)="question()" title="Sponsored content information" class="support-icon">help</mat-icon>
```
Comment (vikaspotluri123) [inline src/app/semester-home/semester-home.component.html]: I'd like a more descriptive method title - e.g. `showSponsorExplainationModal()`

File: src/app/semester-home/semester-home.component.ts [hunk 2]
```diff
@@ -42,4 +45,14 @@ export class SemesterHomeComponent implements OnInit {
...
 	}
+
+	question(): void {
+		this.dialog.open(PopupComponent, {
+			data: {
+				title: 'Sponsored Content Information',
+				text: 'Our partners help pay to keep Aggie Gradebook free for you. '
+							 + 'We\'ll keep this content minimal and ensure that it\'s always educationally relevant.', button: 'Ok'
+				}
+		}).afterClosed().toPromise();
```
Comment (vikaspotluri123) [inline src/app/semester-home/semester-home.component.ts]: I don't think you need this if it's just an alert 🤔

Comment (joshcosta) [reply src/app/semester-home/semester-home.component.ts]: Which part are you referring to? All of those lines?

Comment (vikaspotluri123) [reply src/app/semester-home/semester-home.component.ts]: Yep, the .afterClosed().toPromise() part

**Comment by ramsays** [comment]
On my machine the link gives a 404. How are we handling the external link?

**Comment by vikaspotluri123** [comment]
@joshcosta can you create a PR in server with the corresponding changes?
@ramsays we have a [redirects file](https://github.com/gradebook/server/blob/92943550185ac3db3cfdf574c92c53694f508a43/lib/web/redirects.js) that's going handle forwarding to the sponsor site

**Comment by joshcosta** [comment]
@vikaspotluri123 should I include "client" in the PR?

**Comment by vikaspotluri123** [comment]
No, the client will only be updated when releasing a new version of the server. The only reason for updating the client then is to be able to go back in history and have a functioning server+client combo

The logic for this is a bit iffy, so I'm open to new ideas 😄

**Pull request merged**